features/load-rest.feature

```diff
@@ -2,7 +2,8 @@ Feature: Manage WordPress through endpoints locally
 
   Background:
     Given a WP install
-    And I run `wp plugin install rest-api --activate`
+	When I run `wp core version`
+	Then STDOUT should be a version string >= 4.7
```

Comment:
A caution: this .editorconfig file is useless to me. It is also a hidden requirement and functionality as it is not documented in the coding standards. My editors (e.g. VS Code, notepad++, and vi) do not use such a file. On searching now, it is possible to install a 3rd party extension in VS Code for it to work. However, if that is needed, then such needs to be documented in the coding standards.